index.html

```diff
@@ -6,6 +6,10 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
     <link rel="stylesheet" href="assets/style.css">
+    <script src="assets/luxon.min.js"></script>
+    <script src="assets/script.js"></script>
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/js/bootstrap.bundle.min.js" integrity="sha384-MrcW6ZMFYlzcLA8Nl+NtUVF0sA7MsXsP1UyJoMp4YLEuNSfAP+JcXn/tWtIaxVXM" crossorigin="anonymous"></script>
```

Comment:
- [ ] It looks like exist confusion on this point, the instruction asks you to add these `<script>` lines just **above the body closing tag** and maybe you can ask 'why is this important?' well, I could explain to you this but [this article](https://hackinbits.com/interview-questions/html/why-script-tags-should-be-placed-at-the-end-of-body-tag#:~:text=Why%20tags%20should%20be,the%20end%20of%20body%20tag%3F&text=So%2C%20when%20the%20browser%20encounters,take%20more%20time%20to%20load.) is better than me, I invite you to change again this position. 🐶 
- Just change this and you'll be on the other side. 🐱
![image](https://user-images.githubusercontent.com/20563252/124998909-d1deba00-e012-11eb-9515-cc507c40f0d7.png)